modules/svg/register_types.cpp

```diff
@@ -45,5 +47,6 @@ void register_svg_types() {
 }
 
 void unregister_svg_types() {
+	memdelete(image_loader_svg);
```

Comment:
I'm not sure why you removed this? If you allocate memory, you also need to free it, otherwise it's a leak.

Reply:
Attending to this.